cli/module_generate/scripts/generate_stubs.go

```diff
@@ -80,10 +80,22 @@ func setGoModuleTemplate(clientCode string, module common.ModuleInputs) (*common
 	}
 	var functions []string
 	ast.Inspect(node, func(n ast.Node) bool {
+		if typeSpec, ok := n.(*ast.TypeSpec); ok {
+			if _, ok := typeSpec.Type.(*ast.StructType); ok {
+				if strings.Contains(typeSpec.Name.Name, "Client") {
```

Comment:
As for why I'm looking specifically for Clients, it's because of how I've noticed Go introduces subclasses (I don't know the proper terms in Go). In the client code, the code introduces all subclasses as `<subclass>Client`.